rollup.config.js

```diff
@@ -29,6 +30,48 @@ export default [
                 name: 'posthog',
             },
         ],
+        plugins: [
+            ...plugins,
+            copy({
+                targets: [
+                    {
+                        src: [
+                            'node_modules/rrweb/dist/record/rrweb-record.min.js',
+                            'node_modules/rrweb/dist/plugins/console-record.min.js',
+                        ],
+                        file: 'dist/recorder.js',
+                    },
+                    {
+                        src: 'node_modules/rrweb/dist/record/rrweb-record.min.js.map',
+                        dest: 'dist/',
+                        rename: 'recorder.js.map',
+                    },
+                    {
+                        src: 'node_modules/rrweb/dist/plugins/console-record.min.js.map',
+                        dest: 'dist/',
+                        rename: 'console-record.min.js.map',
+                    },
+                    {
+                        src: 'dist/recorder.js',
+                        dest: 'dist/',
+                        transform: (contents) =>
+                            contents.toString().replace('rrweb-record.min.js.map', 'recorder.js.map'),
```

Comment:
Not sure if this line worked.... I looked in the code generated and the maps seemed not quite right.